src/pages/List.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { FirestoreCollection } from 'react-firestore';
+import Loading from '../components/Loading';
+
+const List = () => {
+  return (
+    <section>
+      <FirestoreCollection
+        // Specify the path to the collection you're pulling data from
+        path="items"
+        // Sort the data
+        sort="name"
+        // isLoading = is a Boolean that represents the loading status for the firebase query. true until an initial payload from Firestore is received.
+        // data = an Array containing all of the documents in the collection. Each item will contain an id along with the other data contained in the document.
+        render={({ isLoading, data }) => {
+          return isLoading ? (
+            <Loading />
+          ) : (
+            <main>
```

Comment:
If there aren't any styles, do we need this wrapping element? I usually try to keep `main` as the dominant element of a page, but looking at the markup, we'd essentially have:

```html
<body>
  <main>
    <form></form>
    <section>
	  <main></main>
	</section>
  </main>
</body>
```

If we really needed the wrapping element, I think `div` would be fine as it would merely be for styling. Screen readers would pickup the `ul` element as something of interest and we can also use `tabindex` if we want to have greater control regarding keyboard nav. But going off the code as it is, wrapping the `ul` element seems unnecessary.

Reply:
No way. This was just a late-night oversight. Fixing now!

Reply:
I hear you 😂